examples/translations/en_es.json

```diff
@@ -0,0 +1,440 @@
+{
+    "en": {
+        "translation": {
+            "DeployKeys": {
+                "title": "Kiva",
+                "authAgency": "Kiva HQ",
+                "authAgencyAcronym": "Kiva"
+            },
+            "Standard": {
+                "back": "Back",
+                "accept": "Accept",
+                "verify": "Verify",
+                "verifying": "Verifying",
+                "continue": "Continue",
+                "poweredBy": "Powered by",
+                "next": "Next",
+                "select": "Select",
+                "goBack": "Go Back"
+            },
+            "Fingers": {
+                "finger": "finger",
+                "right_thumb": "Right Thumb",
+                "right_index": "Right Index",
+                "right_middle": "Right Middle",
+                "right_ring": "Right Ring",
+                "right_little": "Right Little",
+                "left_thumb": "Left Thumb",
+                "left_index": "Left Index",
+                "left_middle": "Left Middle",
+                "left_ring": "Left Ring",
+                "left_little": "Left Little",
+                "right_thumb_full": "Right Thumb",
+                "right_index_full": "Right Index {{finger}}",
+                "right_middle_full": "Right Middle {{finger}}",
+                "right_ring_full": "Right Ring {{finger}}",
+                "right_little_full": "Right Little {{finger}}",
+                "left_thumb_full": "Left Thumb",
+                "left_index_full": "Left Index {{finger}}",
+                "left_middle_full": "Left Middle {{finger}}",
+                "left_ring_full": "Left Ring {{finger}}",
+                "left_little_full": "Left Little {{finger}}"
+            },
+            "PII": {
+                "firstName": "First Name",
+                "lastName": "Last Name",
+                "birthDate": "Date of birth",
+                "momsName": "Mother's First Name",
```

Comment:
this is usually `motherName` and `fatherName`

Reply:
Sure